src/Features/Core/Portable/ExtractMethod/MethodExtractor.VariableSymbol.cs

```diff
@@ -126,7 +126,7 @@ public int CompareTo(ParameterVariableSymbol other)
                     return 0;
                 }
 
-                var compare = CompareTo((IMethodSymbol)_parameterSymbol.ContainingSymbol, (IMethodSymbol)other._parameterSymbol.ContainingSymbol);
+                var compare = CompareMethodParameters((IMethodSymbol)_parameterSymbol.ContainingSymbol, (IMethodSymbol)other._parameterSymbol.ContainingSymbol);
```

Comment:
same here, I can't assume parameter symbol always belong to method symbol... we have this kind assumption all over the places.